nyc_buildings/nyc_buildings.ipynb

```diff
@@ -71,20 +131,38 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "cats = list(ddf.type.value_counts().compute().iloc[:10].index.values) + ['unknown']\n",
-    "ddf['type'] = ddf.type.replace({None: 'unknown'})\n",
-    "ddf = ddf[ddf.type.isin(cats)]\n",
-    "ddf['type'] = ddf['type'].astype('category').cat.as_known()\n",
+    "cats = list(ddf['type'].value_counts().compute().iloc[:10].index.values) + ['unknown']\n",
     "\n",
-    "with ProgressBar():\n",
-    "    ddf = ddf.build_sindex().persist()"
+    "ddf['type'] = ddf['type'].replace({None: 'unknown'})\n",
+    "ddf = ddf[ddf['type'].isin(cats)]\n",
+    "ddf['type'] = ddf['type'].astype('category').cat.as_known()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can view each category separately with a selector widget:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "ddf.hvplot.polygons(rasterize=True, tiles='CartoLight', groupby='type', aggregator='any')"
```

Comment:
Before, in Jim's example, the categories in the widget were alphabetically sorted. It's no longer the case:

![image](https://github.com/user-attachments/assets/0f82d797-12a0-497c-90ce-3fc874c71760)

Jim's code used an intermediate `HoloMap`, this object has a `sort` keyword that is True by default:
![image](https://github.com/user-attachments/assets/d06b6196-c999-4e72-a260-067b0c9e727d)

In hvPlot, `groupby=True` leads internally to applying the `groubpy` operation method of `hv.Dataset`. It's called with `dynamic=True` (the default) which means a DynamicMap is returned and not a HoloMap, loosing that default sorting behavior.

We need to open an issue on hvPlot I think.

Reply:
What would the Issue address? A way to alphabetize a hvPlot `groupby` operation?

Reply:
Yes, but more generally a way to declare that the values of the grouped dimension(s) should be sorted. It should be possible to come up with a small example to reproduce this.

In our case, we could ignore this, or preprocess the dataset in the notebook to sort it by category.

Reply:
OK. I can find a way to create the Issue. 

In this particular example however, I think we can ignore it. I noticed that the final plot that didn't use a `groupby` operation had the categories sorted:

![image](https://github.com/user-attachments/assets/0961523d-e52c-4026-a6f3-ac4ff4174761)

Reply:
To be clear, I've contributed to this example, but it's from @philippjfr , not me! :-)